internal/runbits/dependencies/changesummary_test.go

```diff
@@ -11,49 +11,78 @@ import (
 )
 
 func TestChangeSummary(t *testing.T) {
-	artifact1 := artifact.Artifact{
-		Name: "Package 1", Version: ptr.To("1.0"), ArtifactID: "1",
-		Dependencies: []artifact.ArtifactID{"2", "3"}}
-	artifact2 := artifact.Artifact{
-		Name: "Dependency 1", Version: ptr.To("2.0"), ArtifactID: "2",
-		Dependencies: []artifact.ArtifactID{"4", "5"}}
-	artifact3 := artifact.Artifact{
-		Name: "Dependency 2", Version: ptr.To("3.0"), ArtifactID: "3",
-		Dependencies: []artifact.ArtifactID{"4", "6"}}
-	artifact4 := artifact.Artifact{
-		Name: "Common recursive dependency", Version: ptr.To("4.0"), ArtifactID: "4",
-		Dependencies: nil}
-	artifact5 := artifact.Artifact{
-		Name: "Recursive dependency 1", Version: ptr.To("5.0"), ArtifactID: "5",
-		Dependencies: nil}
-	artifact6 := artifact.Artifact{
-		Name: "Recursive dependency 2", Version: ptr.To("6.0"), ArtifactID: "6",
-		Dependencies: []artifact.ArtifactID{"7"}}
-	artifact7 := artifact.Artifact{
-		Name: "Recursive dependency 3", Version: ptr.To("7.0"), ArtifactID: "7",
-		Dependencies: nil}
-
-	artifact8 := artifact.Artifact{
-		Name: "Package 2", Version: ptr.To("1.1"), ArtifactID: "8",
-		Dependencies: []artifact.ArtifactID{"9", "3"}}
-	artifact9 := artifact.Artifact{
-		Name: "Dependency 1", Version: ptr.To("2.1"), ArtifactID: "9",
-		Dependencies: []artifact.ArtifactID{"10", "5"}}
-	artifact10 := artifact.Artifact{
-		Name: "Common recursive dependency", Version: ptr.To("4.1"), ArtifactID: "10",
-		Dependencies: nil}
+	a1DependsOn2n3 := artifact.Artifact{
```

Comment:
I went cross-eyed reading this test.

Reply:
lmao.. yeah I we cross-eyed trying to understand it. Before it just said `artifact1` and I had a hard time grokking from that how the test was functioning.